src/controllers/features/limited/addressCorrespondanceSelectorController.ts

```diff
@@ -74,9 +76,11 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
                 addressOption
             });
         } else {
+            const applicantDetails = acspData.applicantDetails || {};
```

Comment:
I think this is not required here again as you have got the value outside if above.